### DISCUSSION TITLE
Dump CSM Inati estimation

### DISCUSSION THREAD

**Discussion by SoftVision17** [gadgetron/gadgetron#1115] [Q&A]
Hi all, 

Please I want to know if gadgetron have a property to dump coils estimation.
 
thank you 

![coils_estm](https://user-images.githubusercontent.com/100497592/174580027-4f723528-84cf-4bb4-864e-6ef391d127ce.PNG)

**Comment by valeryozenne** [comment]
Hi

I guess, if you set the debug_folder property like /tmp/ or /home/xxxx/ you should get some data in analyze format (readable in python or Matlab) named ref_coil_map_after_filtering_encoding_0_MAG.hdr et .img including PHASE, REAL and IMAG part.  

https://github.com/gadgetron/gadgetron/blob/master/gadgets/mri_core/generic_recon_gadgets/GenericReconCartesianGrappaGadget.cpp#L126

Best regards,
Valéry

**Reply by SoftVision17** [reply]
Hi,

Thank you for your response .

I set the value for the debug_folder to /tmp/myfolder/ , but nothing was generated, I don't know if it's normal or I need to set another parameter to get the data.

**Comment by xueh2** [comment]
I thought after fixing the permission, this issue is solved?
